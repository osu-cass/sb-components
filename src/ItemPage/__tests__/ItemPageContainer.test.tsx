import * as React from "react";
import * as ReactDOM from "react-dom";
import { shallow, mount, render } from "enzyme";
import { ItemPageContainer } from "@src/index";
import {
  itemPageContainerPropsMock,
  itemPageContainerPropsNoItemMock
} from "@mocks/ItemPage/mocks";

describe("ItemPageContainer", () => {
  it("renders correctly", () => {
    const wrapper = shallow(
      <ItemPageContainer {...itemPageContainerPropsMock} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it("It renders correctly when there is no item", () => {
    const wrapper = shallow(
      <ItemPageContainer {...itemPageContainerPropsNoItemMock} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
