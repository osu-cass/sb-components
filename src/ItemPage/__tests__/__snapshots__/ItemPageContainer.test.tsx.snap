// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemPageContainer It renders correctly when there is no item 1`] = `
<LoadingOverlay
  loading={true}
>
  <div
    className="container item-page"
  />
</LoadingOverlay>
`;

exports[`ItemPageContainer renders correctly 1`] = `
<LoadingOverlay
  loading={true}
>
  <div
    className="container item-page"
  />
</LoadingOverlay>
`;
